Tasks/Vadeika/Task2.py

```diff
@@ -0,0 +1,111 @@
+print(f'"Вы ввели цифру : {select}"')
+dict_val = {
+    '00': ['ноль минут', 'двенадцать часов', 'час', 'первого'],
+    '01': ['одна минута', 'один час', 'два', 'второго'],
+    '02': ['две минуты', 'два часа', 'три', 'третьего'],
+    '03': ['три минуты', 'три часа', 'четыре', 'четвертого'],
+    '04': ['четыре минуты', 'четыре часа', 'пять', 'пятого'],
+    '05': ['пять минут', 'пять часов', 'шесть', 'шестого'],
+    '06': ['шесть минут', 'шесть часов', 'семь', 'седьмого'],
+    '07': ['семь минут', 'семь часов', 'восемь', 'восьмого'],
+    '08': ['восемь минут', 'восемь часов', 'девять', 'девятого'],
+    '09': ['девять минут', 'девять часов', 'десять', 'десятого'],
+    '10': ['десять минут', 'десять часов', 'одиннадцать', 'одиннадцатого'],
+    '11':
+    ['одинндцать минут', 'одиннадцать часов', 'двенадцать', 'двенадцатого'],
+    '12': ['двенадцать минут', 'двенадцать часов', 'час', 'первого'],
+    '13': ['тринадцать минут', 'один час', 'два', 'второго'],
+    '14': ['четырнадцать минут', 'два часа', 'три', 'третьего'],
+    '15': ['пятнадцать минут', 'три часа', 'четыре', 'четвертого'],
+    '16': ['шестнадцать минут', 'четыре часа', 'пять', 'пятого'],
+    '17': ['семнадцать минут', 'пять часов', 'шесть', 'шестого'],
+    '18': ['восемнадцать минут', 'шесть часов', 'семь', 'седьмого'],
+    '19': ['девятнадцать минут', 'семь часов', 'восемь', 'восьмого'],
+    '20': ['двадцать минут', 'восемь часов', 'девять', 'девятого'],
+    '21': ['двадцать одна минута', 'девять часов', 'десять', 'десятого'],
+    '22':
+    ['двадцать две минуты', 'десять часов', 'одиннадцать', 'одиннадцатого'],
+    '23':
+    ['двадцать три минуты', 'одиннадцать часов', 'двенадцать', 'двенадцатого'],
+    '24': ['двадцать четыре минуты'],
+    '25': ['двадцать пять минут'],
+    '26': ['двадцать шесть минут'],
+    '27': ['двадцать семь минут'],
+    '28': ['двадцать восемь минут'],
+    '29': ['двадцать девять минут'],
+    '30': ['половина'],
+    '31': ['тридцать одна минута'],
+    '32': ['тридцать две минуты'],
+    '33': ['тридцать три минуты'],
+    '34': ['тридцать четыре минуты'],
+    '35': ['тридцать пять минут'],
+    '36': ['тридцать шесть минут'],
+    '37': ['тридцать семь минут'],
+    '38': ['тридцать восемь минут'],
+    '39': ['тридцать девять минут'],
+    '40': ['сорок минут'],
+    '41': ['сорок одна минута'],
+    '42': ['сорок две минуты'],
+    '43': ['сорок три минуты'],
+    '44': ['сорок четыре минуты'],
+    '45': ['без пятнадцати минут'],
+    '46': ['без четырнадцати минут'],
+    '47': ['без тринадцати минут'],
+    '48': ['без двенадцати минут'],
+    '49': ['без одиннадцати минут'],
+    '50': ['без десяти минут'],
+    '51': ['без девяти минут'],
+    '52': ['без восьми минут'],
+    '53': ['без семи минут'],
+    '54': ['без шести минут'],
+    '55': ['без пяти минут'],
+    '56': ['без четырех минут'],
+    '57': ['без трех минут'],
+    '58': ['без двух минут'],
+    '59': ['без одной минуты']
+}
```

Comment:
not bad. but could be better, you have many duplicates of the values, -5 points for not optimal data structure